src/dist.rs

```diff
@@ -315,6 +345,78 @@ impl Dist {
     }
 }
 
+#[cfg(feature = "scss")]
+fn scss(
+    static_dir: &std::path::Path,
+    dist_dir: &std::path::Path,
+    options: sass_rs::Options
+) -> Result<()> {
+    use walkdir::{DirEntry, WalkDir};
+
+    fn is_scss(path: &std::path::Path) -> bool {
+        matches!(
+            path.extension().map(|x| x.to_str()).flatten(),
+            Some("sass") | Some("scss")
+        )
+    }
+
+    fn should_ignore(entry: &DirEntry) -> bool {
+        entry
+            .file_name()
+            .to_str()
+            .map(|x| x.starts_with("_"))
+            .unwrap_or(false)
+    }
+
+    log::trace!("Copying static directory into dist directory");
+    let walker = WalkDir::new(&static_dir).into_iter();
+    for entry in walker
+        .filter_map(|x| match x {
+            Ok(x) => {
+                if !x.path().is_dir() && !should_ignore(&x) {
+                    Some(x)
+                } else {
+                    None
+                }
+            },
+            Err(_err) => {
+                log::error!("could not walk into directory: `{}`", &static_dir.display());
+                None
+            }
+        })
+    {
+        let file_path = entry.path();
+
+        if is_scss(file_path) {
+            let css_path = dist_dir
+                .join(&file_path.strip_prefix(&static_dir).unwrap())
+                .with_extension("css");
+
+            match sass_rs::compile_file(file_path, options.clone()) {
+                Ok(css) => {
+                    let _ = fs::create_dir_all(css_path.parent().unwrap());
```

Comment:
We also need to create the parent for the `dist_path`, duplicate

Reply:
Make sense, we do this because we do not need to create anything if `sass_rs::compile_file` return an error.